server/locale/DE/GPL/mutations/gpl_eqz.js

```diff
@@ -1,9 +1,79 @@
 import Logo from '../../../../message/logos';
 import { xSmallFallback, textWrap, messageLogoWidth, altNoWrap, setLogoTop } from '../../../../message/mediaQueries';
 import { addPeriod, xSmallWrap } from './mediaQueries';
-import { textLogoMutations } from '../../../../message/logoMutations';
+import { flexLogoMutations, textLogoMutations } from '../../../../message/logoMutations';
+
+const headlineBreaks = [
+    {
+        sizes: ['xsmall'],
+        breaks: ['Ratenzahlung']
+    },
+    {
+        sizes: ['medium'],
+        breaks: ['Sie in', 'Einkäufen']
+    }
+].reduce((acc, item) => {
+    const { sizes, breaks } = item;
+    sizes.forEach(size => {
+        acc.push({
+            tag: size,
+            br: breaks
+        });
+    });
+    return acc;
+}, []);
+
+const flex = [
+    [
+        'default',
+        {
+            logo: Logo.PP_PAYPAL.WHITE,
+            headline: [
+                {
+                    tag: 'xsmall'
+                },
+                {
+                    tag: 'medium'
+                }
+            ],
+            disclaimer: ['default']
+        }
+    ],
+    [
+        'ratio:20x1',
+        {
+            styles: [
+                `
+                @media (max-width: 350px) {
+                    .message__messaging  {
+                        padding-right: 12%;
+                    }
+                }
+                `
+            ],
+            headline: [...headlineBreaks]
+        }
+    ],
+    [
+        'ratio:8x1',
+        {
+            styles: [
+                `
+                @media (max-width: 350px) {
+                    .message__messaging  {
+                        padding-right: 12%;
+                    }
+                }
+                `
+            ],
```

Comment:
See above explanation. 
```suggestion
            styles: [
                `.message__headline > .tag--medium > span > span:last-child::after {
                    content: '.'
                }`
            ],
```